### PR TITLE
:pencil2: fixed get me in mail sending feature and updated tokens

### PR DESCRIPTION
**Pull request by DhruvDave12**
Fixed the bug because of which mail weren't being send to the users and removed slack feature

### CODE REVIEW

File: constants.js [hunk 1]
```diff
@@ -2,5 +2,5 @@ exports.token = process.env.SECRET
 exports.slack = process.env.SLACK_TOKEN
 exports.webhookURL = process.env.INVITE_CHANNEL_WEBHOOK
 exports.glitch = process.env.GLITCH_SECRET
-exports.selfEmail = 'IIITV Coding Club <codingclub@iiitv.ac.in>'
+exports.selfEmail = 'IIITV Coding Club <davedhruv1201@gmail.com>'
```
Comment (neeraj3029) [inline constants.js:5]: change it to CC's email

File: public/client.js [hunk 2]
```diff
@@ -81,16 +81,26 @@ $.put = function (url, data, callback, type) {
...
     fetch(`https://api.github.com/search/users?q=${username}`)
       .then(res => res.json())
       .then((out) => {
-        if (out.total_count === 1) {
-          fetch(`/sendmail/${username}/${email}`)
+        // find the user with username in the api if not found then send error
+        console.log("OUTPUT OF GITHUB API: ", out);
+        var flag = false;
+
+        for(let i=0; i<out.total_count; i++){
```
Comment (neeraj3029) [inline public/client.js:96]: Can you share a sample response here? Has the response type changed from object to object[] ?

Comment (neeraj3029) [inline public/client.js:96]: Use `Array.filter` to filter out required users and remove all the flag logic

File: public/client.js [hunk 3]
```diff
@@ -109,20 +119,56 @@ $(function () {
...
+            // eslint-disable-next-line
+            UIkit.notification({
+              message: '<span class=\'uk-text-small\' uk-icon=\'icon: warning\'>Username not found.</span>',
+              status: 'danger',
+              pos: 'top-center',
+              timeout: 1000
+            })
+          }
         }
+        // if (out.total_count === 1) {
```
Comment (neeraj3029) [inline public/client.js:135]: Remove commented code and logs

**Review by neeraj3029 — CHANGES_REQUESTED** [review]

**Comment by neeraj3029** [comment]
Why do we have lockfile changes without `package.json` updates?

File: public/client.js (hunk 2 above)
Comment (aashutoshrathi) [reply public/client.js:96]: yup! seems like that
https://api.github.com/search/users?q=linus

Comment (aashutoshrathi) [reply public/client.js:96]: Also, I think in our case, we don't need to iterate, ideally, there should be either 0 or 1 entries in the response.

Comment (aashutoshrathi) [reply public/client.js:96]: Wait I'm not even sure if we need search API, I think our use case will be solved using User API
Eg: https://api.github.com/users/aashutoshrathi

**Review by aashutoshrathi — CHANGES_REQUESTED** [review]
There are some comments by @neeraj3029 that needs to be resolved, plus code-quality is bad in general in this repo, so when you get time let's clean that up too.
Once all that is done, let me know I can provide one of you people the access to Glitch deployment (ideally someone with access to CC Email, since it has details linked to that in env)

File: public/client.js (hunk 2 above)
Comment (DhruvDave12) [reply public/client.js:96]: okay sir i will update the right now i checked on entering a username an array of object with username in different combination of lowercase and uppercase were coming so i will update the API to get user one

Comment (DhruvDave12) [reply public/client.js:96]: yes sir sure will update the code

**Comment by DhruvDave12** [comment]
I have updated the github api to fetch the user directly, removed the comments and unnecessary logs, and also fixed one toast issue that send the message of "Thank you, check your inbox" only if the username exists.

Remaining changes like
1. Updating the CC mail.
2. Putting the credentials of CC.
3. Improving further code quality

This all will be done latest by tomorrow.

**Review by Im-Madhur-Gupta — APPROVED** [review]
LGTM

**Pull request merged**